src/main/java/org/kainos/ea/db/DatabaseConnector.java

```diff
@@ -0,0 +1,39 @@
+package org.kainos.ea.db;
+
+import java.io.FileInputStream;
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+import java.util.Properties;
+
+public class DatabaseConnector {
+    private static Connection conn;
+
+    public Connection getConnection() throws SQLException {
+        String user, password, host, name;
+
+        if(conn != null && !conn.isClosed()) { return conn; }
+
+        try( FileInputStream propsStream = new FileInputStream("db.properties");){
```

Comment:
This can be cleaned up to `try(FileInputStream propsStream = new FileInputStream("db.properties")){`